mockgen.hcl

```
@@ -0,0 +1,12 @@
+description = "GoMock is a mocking framework for the Go programming language."
+binaries = ["mockgen"]
+test = "mockgen --test"
```

I think this returns an error: `flag provided but not defined: -test`

```suggestion
test = "mockgen --version"
```